app/models/reporting/processed_report.rb

```diff
@@ -0,0 +1,60 @@
+module Reporting
+  class ProcessedReport
+    def initialize(day_zero: Time.zone.now.to_date)
+      @day_zero = day_zero
+      @slug = 'processed_report'
+    end
+
+    def id
+      @slug
+    end
+
+    def headers
```

Comment:
One thing I was thinking is that we could have a base class `BaseReport` and move the common methods to it:

```ruby
module Reporting
    class BaseReport
        def initialize(slug)
            @slug = slug
        end

        def id
            @slug
        end

       def headers
          # ...
       end
 
       def table
           # override
       end

      def rows
          # ...
      end
end

```

There is a little bit of duplication here that we could get rid of with this if we wanted - not sure its worth it though.

Reply:
👍